website/docs/r/federated_settings_org_role_mapping.html.markdown

```diff
@@ -49,7 +49,7 @@ resource "mongodbatlas_federated_settings_org_role_mapping" "org_group_role_mapp
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
-* `id` - Unique 24-hexadecimal digit string that identifies this role mapping.
```

Comment:
even if `role_mapping_id` is exposed, `id` still would be exposed

Reply:
Correct, but the description is misleading since the `id` is a base64 encoded string. 
I don't see any reason for the user to use the `id` and would prefer to keep it undocumented in the docs.

Reply:
Correct me if I am wrong but `id` (base64 encoded) can still be used to import the resource right? If so i would keep it here and change de description of it

Reply:
and most likely import method should be modified to set the new attribute role_mapping_id with the correct value

Reply:
No, `id` cannot be used directly. The [data source expects `role_mapping_id`](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/CLOUDP-248288_role_mapping_id/internal/service/federatedsettingsorgrolemapping/data_source_federated_settings_org_role_mapping.go#L77). You would have to create an equivalent `conversion.DecodeStateID` in terraform for it to work :S

Reply:
The import method parses the `role_mapping_id` correctly; however, until this PR, it was never set on the resource other than in the encoded `id` field